frontend/src/components/atoms/Shared/TextButton/TextButton.tsx

```diff
@@ -0,0 +1,17 @@
+import {Button} from "@material-ui/core";
+import {Link} from "react-router-dom";
+
+type TextButtonProps = {
+    readonly text: string;
+    readonly onLink: string;
+}
+
+const TextButton = ({text, onLink}: TextButtonProps) => {
```

Comment:
To też raczej nie ma sensu. Mamy Button z materiala.

Reply:
to jak? :P